pkg/controller.v1/common/job.go

```diff
@@ -45,32 +45,6 @@ func (jc *JobController) DeletePodsAndServices(runPolicy *apiv1.RunPolicy, job i
 	return nil
 }
 
-func (jc *JobController) cleanupJobIfTTL(runPolicy *apiv1.RunPolicy, jobStatus apiv1.JobStatus, job interface{}) error {
```

Comment:
Which code is referencing this function?

Reply:
@johnugeorge I don't find any code is referencing the cleanupJobIfTTL function. And I found that the cleanupJobIfTTL function is same with the CleanupJob function( before I edit), so I removed it.

Reply:
LGTM, it seems that the func is not used now.